modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/legacy_appeals_controller.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'caseflow/service'
+require 'common/exceptions'
+
+class AppealsApi::V2::DecisionReviews::LegacyAppealsController < AppealsApi::ApplicationController
+  SSN_REGEX = /^[0-9]{9}$/.freeze
+
+  skip_before_action(:authenticate)
+  before_action :validate_headers, only: %i[index]
+
+  EXPECTED_HEADERS = %w[X-VA-SSN X-VA-File-Number].freeze
+
+  UNUSABLE_RESPONSE_ERROR = {
+    errors: [
+      {
```

Comment:
We should already have a common exception for a bad gateway. Let's use that instead. Our consumers won't necessarily have any context for what caseflow is. If _we_ want to know there was an error from caseflow (and we do), we should log it but not bubble up the service that failed to the consumer.

Reply:
OK I'll look into this. We use it in contestable issues I believe.

Reply:
I'm going to rewire this design. I reused it and realize now that I probably shouldn't have.

Reply:
reworked for something a little more simple.